metrics/lib/common.bash

```diff
@@ -29,6 +29,12 @@ KSM_PAGES_SHARED="${KSM_BASE}/pages_shared"
 KSM_AGGRESIVE_PAGES=1000
 KSM_AGGRESIVE_SLEEP=50
 
+declare -A registries
+registries[ubuntu]= \
+"docker.io/library
+public.ecr.aws/ubuntu
```

Comment:
@devimc I think we need to remove `ubuntu` from here

Reply:
@devimc the name of the image should be `public.ecr.aws/lts/ubuntu`

Reply:
@devimc now it's `public.ecr.aws/lts`, but we need `public.ecr.aws/lts/ubuntu`

Reply:
I don't think so, ptal https://gallery.ecr.aws/lts/ubuntu

**Usage**

```
Docker CLI

Launch this image locally:

docker run -d --name ubuntu -e TZ=UTC public.ecr.aws/lts/ubuntu:20.04_stable
```

Reply:
@devimc yes, of course. Missed that the `/ubuntu` is still appended. Sorry for the noise.